decred/decred/crypto/crypto.py

```diff
@@ -13,10 +13,10 @@
 from blake256.blake256 import blake_hash
 import nacl.secret
 
-from tinydecred.util.encode import ByteArray
-from tinydecred.crypto import rando
-from tinydecred.crypto.secp256k1.curve import curve as Curve, PublicKey, PrivateKey
-from tinydecred.util import encode
+from decred.util import encode
+from decred.util.encode import ByteArray
+from . import rando
```

Comment:
The relative imports are working?

Reply:
Tests are passing and the wallet works, so I'd say yes, they are.